admin_scripts/ubuntu/ubuntu_16.04/oracle_java.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+# http://askubuntu.com/questions/430434/replace-openjdk-with-oracle-jdk-on-ubuntu
+
+add-apt-repository ppa:webupd8team/java -y
+apt-get update
+
+echo "oracle-java6-installer shared/accepted-oracle-license-v1-1 select true" | sudo debconf-set-selections
+apt-get install oracle-java6-installer -y
```

Comment:
As before. Is this still useful? (Java _6_? That hasn't had security support for years, has it?)